### DISCUSSION TITLE
[Landing]: Update WebXDAO website design

### DISCUSSION THREAD

**Discussion by mkubdev** [WebXDAO/WebXDAO.github.io#184] [Show and tell]
# Introduction

![image](https://user-images.githubusercontent.com/50140834/189951719-4c681c6b-77a6-4ccc-8147-a24f8e7c08a2.png)

This could be the next hero component of the landing page ! What do you think?

> 💡 : This is copycat content from discord. I'm testing GitHub Discussions.

# Motion / Controls

Added [`framer-motion`](https://www.framer.com/docs/) package to play with custom motion :

![framer-motion](https://user-images.githubusercontent.com/50140834/190118889-c40e231d-d068-4c38-a5a4-b4bf26f0e857.gif)

# Responsivness

Added responsive class + Navbar breakpoint :
> The illustration is temporary

![Responsive](https://user-images.githubusercontent.com/50140834/190192024-a96e1d3d-6846-4199-9bc9-899edf7f9daf.gif)

# Mobile menu

> EDIT: Added webxdao white logo

![hehe5](https://user-images.githubusercontent.com/50140834/190402610-b1984a81-dc87-43bc-9dad-e4ff1fa25341.gif)

# Final

![image](https://user-images.githubusercontent.com/50140834/190584226-adba72bd-b1e2-467e-b67f-3b5944e66860.png)

**Comment by shuklaritvik06** [comment]
For me this looks good but why u removed that hacktobefest one @mkubdev ?

**Reply by mkubdev** [reply]
@shuklaritvik06 It's encapsulated into a component don't worry! I removed it for the screen record because it needs more tweaking to have a fully responsive layout. 😊

**Reply by shuklaritvik06** [reply]
Ohhhkk cool

**Reply by mkubdev** [reply]
@shuklaritvik06 hehe added in # Final chapter

**Comment by Simperfy** [comment]
Looks Great 🔥

**Reply by mkubdev** [reply]
Thanks @Simperfy ! 🙌

**Comment by AbhinavMV** [comment]
Niceee, 🔥
Maybe we can change the background color, Our logo is not properly visible in this one. What do you think?

**Reply by mkubdev** [reply]
Yes, the logo isn't visible at all 😅 We should try with a white one!

Or using a lighter gradient...

**Reply by AbhinavMV** [reply]
Lighter gradient would be nice or maybe logo with white background 😄

**Reply by mkubdev** [reply]
@AbhinavMV Much better with the white logo!

**Comment by vinzvinci** [comment]
Available on https://github.com/orgs/WebXDAO/projects/9
